web-client/src/views/TrialSessions/TrialLocationEligibleCasesTable.tsx

```diff
@@ -0,0 +1,124 @@
+import { connect } from '@web-client/presenter/shared.cerebral';
+import { CaseIcons } from '@web-client/ustc-ui/Icon/CaseIcons';
+import { CaseLink } from '@web-client/ustc-ui/CaseLink/CaseLink';
+import { Paginator } from '@web-client/ustc-ui/Pagination/Paginator';
+import { focusPaginatorTop } from '@web-client/presenter/utilities/focusPaginatorTop';
+import { sequences, state } from '@web-client/presenter/app.cerebral';
+import React, { useRef } from 'react';
+
+export const TrialLocationEligibleCasesTable = connect(
+  {
+    setTrialLocationPaginatorSequence:
+      sequences.setTrialLocationPaginatorSequence,
+    trialLocationHelper: state.trialLocationHelper,
+    trialLocationPage: state.trialLocationPage,
+  },
+  function TrialLocationEligibleCasesTable({
+    setTrialLocationPaginatorSequence,
+    trialLocationHelper,
+    trialLocationPage,
+  }) {
+    const paginatorTop = useRef(null);
+
+    return (
+      <>
+        <div className="grid-row margin-bottom-2 flex-align-center">
+          <div className="grid-col" ref={paginatorTop}>
+            <Paginator
+              currentPageIndex={trialLocationPage.eligibleCasesPage}
+              totalPages={trialLocationHelper.totalPagesEligible}
+              onPageChange={selectedPage => {
+                setTrialLocationPaginatorSequence({
+                  pageNumber: selectedPage,
+                  pageType: 'eligibleCasesPage',
+                });
+                focusPaginatorTop(paginatorTop);
+              }}
+            />
+            <div className="grid-col-2"></div>
+          </div>
+        </div>
+        <div className="text-right">
+          <span className="text-semibold">Count: </span>
+          {trialLocationHelper.formattedEligibleCases.length}
+        </div>
+        <div className="padding-1"></div>
+        <div className="overflow-x-auto">
+          <div className="minw-tablet-lg">
+            <table
+              aria-label={`${trialLocationPage.currentTab}`}
```

Comment:
Do we need this aria-label? If so, should we add it to the TrialLocationBlockedCasesTable?